src/callbacks.c

```diff
@@ -634,7 +634,11 @@ gboolean toolbar_popup_menu(GtkWidget *widget, GdkEventButton *event, gpointer u
 {
 	if (event->button == 3)
 	{
+#if GTK_CHECK_VERSION(3,22,0)
```

Comment:
The GTK version check and popup menu commands repeat quite often and look very similar.

Could we maybe put them into a helper function, maybe `ui_menu_poopup()` in "ui_utils.c" or so, to not have to repeat this everywhere?
Also we could add a sentence or two to document what happens there.

Reply:
Can we just assume 3.22 IAW [this comment](https://github.com/geany/geany/pull/3178#issuecomment-1128174560)

Reply:
@eht16: Sure, makes sense. Thanks for the hint on the function name and where it should be located, that was really helpful.

@elextr: It would definitely make things simpler, if there is agreement that 3.22 or higher is required. But I believe it is not yet enforced in autotools nor in meson, so I will keep it backward compatible, at least for now, ok?

Reply:
> so I will keep it backward compatible, at least for now, ok?

Sure, was just trying to see if it could be simplified :-)